presto-main/src/main/java/com/facebook/presto/server/security/LdapFilter.java

```diff
@@ -119,7 +124,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
                 context = authenticate(environment);
 
                 if (groupDistinguishedName.isPresent()) {
-                    if (!ldapBinder.checkForGroupMembership(user, groupDistinguishedName.get(), context)) {
+                    if (!checkForGroupMembership(user, groupDistinguishedName.get(), context)) {
```

Comment:
I would just move this throw statement to the method now, instead of checking the returned flag.